support/util/expose.go

```diff
@@ -35,6 +35,11 @@ func ServicePublishingStrategyByTypeByHC(hc *hyperv1.HostedCluster, svcType hype
 	return nil
 }
 
+func IsRouteKASByHC(hc *hyperv1.HostedCluster) bool {
```

Comment:
Could you add unit tests for this function please?

Reply:
Yup, done